LemonWay/Models/KycDoc.php

```diff
@@ -76,4 +76,4 @@ function __construct($node) {
 	}
 }
 
-?>
\ No newline at end of file
+?>
```

Comment:
Thanks for your answer! I have used the GitHub online editor. I have only changed the constant value and added the new one. If you check the file you will see that there is no change with the original version.